src/transaction/log_applier_sql_log.c

```diff
@@ -546,6 +562,11 @@ sl_write_sql (string_buffer & query, string_buffer * select)
   if (ftell (log_fp) >= SL_LOG_FILE_MAX_SIZE)
     {
       log_fp = sl_open_next_file (log_fp);
+
+      if (sql_log_keep_cnt != 0 && sl_Info.curr_file_id >= sql_log_keep_cnt)
```

Comment:
I think `sql_log_keep_cnt` can not be 0, because system parameter will be set to the value larger than 1.
how about just adding the assert ? ( assert (sql_log_keep_cnt > 0))

Reply:
sql_log_keep_cnt != 0 is for the previous spec. I will remove it. But I think that assert() is not needed because the lower and upper range check are already done in the system paramter module.